### ISSUE TITLE
[backstage] security considerations

### ISSUE THREAD

**Issue by jkleinlercher** [suxess-it/sx-cnp-oss#98]
https://backstage.io/docs/auth/

> NOTE: Identity management and the Sign-In page in Backstage is NOT a method for blocking access for unauthorized users. The identity system only serves to provide a personalized experience and access to a Backstage Identity Token, which can be passed to backend plugins. This also means that your Backstage backend APIs are by default unauthenticated. Thus, if your Backstage instance is exposed to the Internet, anyone can access information in the Backstage. You can learn more [here](https://backstage.io/docs/overview/threat-model#integrator-responsibilities).

**Comment by jkleinlercher** [comment]
@phac008 is this issue solved with the new backstage version?

**Comment by phac008** [comment]
yes, currently there is a lot of effort on this topic anyhow, especially coming with MCP integration. will close that for now
